src/store/user/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { ADD_USER, DELETE_USER } from './types';
+
+export const userInitialState = [];
```

Comment:
User initial state should look a bit different, please see the [reference](https://d17btkcdsmqrmh.cloudfront.net/react-fundamentals/docs/module-3/home-task/task#project-structure-requirements)

Reply:
fixed